src/StableRNGs.jl

```diff
@@ -155,4 +155,29 @@ function Random.shuffle!(r::StableRNG, a::AbstractArray)
     return a
 end
 
+# https://github.com/JuliaRandom/StableRNGs.jl/issues/20
+@noinline function Random.randn_unlikely(rng::StableRNG, idx, rabs, x)
```

Comment:
I wonder if we should do the full `randn` definition, or just this `randn_unlikely` path? Moving less code here is nice, but since this seems to be an internal method maybe it will just break again

Reply:
I don't have a strong opinion. My impression is that currently it's easier to just redefine these two functions in StableRNGs and reconsider if additional breakage occurs in some future Julia release. Of course, to be sure that upstream changes won't affect StableRNGs one would have to copy basically the whole `src/normal.jl` file from Random.